content/source/docs/cloud/workspaces/settings.html.md

```diff
@@ -48,13 +48,11 @@ The display name of the workspace.
 
 ### Execution Mode
 
-[remote backend]: /docs/language/settings/backends/remote.html
-
 Whether to use Terraform Cloud as the Terraform execution platform for this workspace.
 
 The default value is "Remote", which instructs Terraform Cloud to perform Terraform runs on its own disposable virtual machines. This provides a consistent and reliable run environment, and enables advanced features like Sentinel policy enforcement, cost estimation, notifications, version control integration, and more.
 
-To disable remote execution for a workspace, change its execution mode to "Local". The workspace will store state, which Terraform can access using the [remote backend][].
+To disable remote execution for a workspace, change its execution mode to "Local". The workspace will store state, which Terraform can access using the [CLI integration](/docs/cli/cloud/index.html).
```

Comment:
(This is in critique of the original wording, so for the purposes of this PR, I understand if you want to ignore it)

This sentence is too vague, I think. As someone who doesn't yet know anything about the various workflows, I don't know what "Terraform can access using the CLI integration" means. 

Something like...(this is worded poorly, hastily, but gets at what I mean)

> To disable remote execution for a workspace, change its execution mode to "Local". This mode is used solely for the CLI-driven run workflow [link]. In this mode, the workspace will be used to store state and variables for using Terraform on a local command line, with the process running on your local machine.

Reply:
Fair - thank you!! I updated :-)